certbot/docs/contributing.rst

```diff
@@ -154,7 +154,7 @@ Running automated integration tests
 
 Generally it is sufficient to open a pull request and let Github and Azure Pipelines run
 integration tests for you. However, you may want to run them locally before submitting
-your pull request. You need Docker and docker-compose installed and working.
+your pull request. You need Docker and docker-compose (v1 or v2) installed and working.
```

Comment:
is it true that v1 still works?

if so, feel free to keep this as is, but i actually think we can probably just delete the `docker-compose` requirement here. we required `docker-compose` here once upon a time, but we actually no longer require it for pebble which is what's being used in the description below and your work in https://github.com/certbot/certbot/pull/9918 exists

Reply:
aha, you're very right. removing!